src/FileAdder/FileAdder.php

```diff
@@ -191,6 +194,20 @@ public function withCustomProperties(array $customProperties)
         return $this;
     }
 
+    /**
+     * Set the metadata.
```

Comment:
Can you change this in `Set the manipulations.`?

Reply:
done